src/test/java/uk/gov/companieshouse/overseasentitiesapi/interceptor/ProcessingInterceptorTest.java

```diff
@@ -112,4 +112,42 @@ void testInterceptorReturnsFalseWhenNoTransactionFound() {
         assertFalse(result);
         assertEquals(HttpServletResponse.SC_INTERNAL_SERVER_ERROR,  mockHttpServletResponse.getStatus());
     }
+
+    @Test
+    void testInterceptorReturnsTrueWhenForceFlagIsTrueAndGetRequest() {
+        MockHttpServletResponse mockHttpServletResponse = new MockHttpServletResponse();
+        Object mockHandler = new Object();
+
+        when(mockHttpServletRequest.getParameter("force")).thenReturn("true");
+        when(mockHttpServletRequest.getMethod()).thenReturn(HttpMethod.GET.name());
+        var result = processingInterceptor.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler);
+
+        assertTrue(result);
+        assertEquals(HttpServletResponse.SC_OK,  mockHttpServletResponse.getStatus());
+    }
+
+    @Test
+    void testInterceptorReturnsTrueWhenForceFlagIsTrueAndPutRequest() {
+        MockHttpServletResponse mockHttpServletResponse = new MockHttpServletResponse();
+        Object mockHandler = new Object();
+
+        when(mockHttpServletRequest.getParameter("force")).thenReturn("true");
+        when(mockHttpServletRequest.getMethod()).thenReturn(HttpMethod.PUT.name());
+        var result = processingInterceptor.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler);
+
+        assertTrue(result);
+        assertEquals(HttpServletResponse.SC_OK,  mockHttpServletResponse.getStatus());
+    }
+
+    @Test
+    void testInterceptorReturnsTrueWhenForceFlagIsFalse() {
```

Comment:
I think the test name needs updating here,  should be ...ReturnsFalse..   instead of ...ReturnsTrue...